aptible/docker-rabbitmq
# Add RabbitMQ 3.7

Adds a bunch of variables since there are many differences between 3.5 and 3.7:

* 3.7 requires an erlang version  only present in Alpine 3.8
* rabbitmqadmin requires a different version
* use of rabbitmqadmin w/ 3.7 requires a flag to ignore untrusted CA, the 3.5 version was manually patched because it had no such flag
* Download locations, file names, file types, and unzip commands are different between installers for 3.5 and 3.7
* Ditto for the delayed message exchange plugin
* Changed the username for testing from "user" to "testuser": the command `rabbitmqctl list_users` now returns a header that contains the string "user", which broke `add_user_if_not_exists ()`

Other things fixed:
 * The "It should be able to publish and retrieve a message" didn't actually publish or retrieve a message
* Useless use of `run; [ "$status" -eq "0" ]` in bats test
* Use library/alpine directly
